docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -434,6 +437,38 @@ from perf counters log. The "exec type" field will contain the implementation ty
 
    MatMul_1800         EXECUTED         layerType: FullyConnected         execType: brgemm_avx512_amx_sparse_I8 realTime (ms): 0.050000  cpuTime (ms): 0.050000
 
+Multi-Threading Optimization
```

Comment:
Please move this section before `Denormals Optimization` as long as it has higher priority.